src/firefly/js/ui/tap/TapViewType.jsx

```diff
@@ -272,6 +273,20 @@ function BasicUI(props) {
         });
     };
 
+    const loadObsCoreMeta = (serviceUrl, obsCoreTableModel) => {
+        const [, obsCoreTable] = obsCoreTableModel?.tableData?.data?.[0];
+        if (!obsCoreTable || !supportsObsCoreMetadataLoad(serviceUrl)) {
+            setObsCoreMetadataModel(undefined); //indicates loading attempt wasn't made
+        }
+        else {
+            setObsCoreMetadataModel({loading: true}); //indicates intermediate state until tableModel is available (Promise is resolved/rejected)
+            const columns = ['obs_collection', 'instrument_name', 'facility_name', 'dataproduct_subtype'].join(',');
```

Comment:
Why not put this list in `loadObsCoreMetadata()`? I think it would be better to push it into the utility since these are standard columns.